test_util/src/pty.rs

```diff
@@ -181,7 +181,7 @@ impl Pty {
     mut condition: impl FnMut(&mut Self) -> bool,
   ) {
     let timeout_time =
-      Instant::now().checked_add(Duration::from_secs(5)).unwrap();
+      Instant::now().checked_add(Duration::from_secs(15)).unwrap();
```

Comment:
Maybe we should only do this on the CI? It's useful to have a shorter timeout when debugging issues.